src/compiler/transpile/run-program.ts

```diff
@@ -50,10 +54,30 @@ export const runTsProgram = async (
     }
   };
 
-  // Emit files that changed
-  tsBuilder.emit(undefined, emitCallback, undefined, false, {
+  const transformers: ts.CustomTransformers = {
     before: [convertDecoratorsToStatic(config, buildCtx.diagnostics, tsTypeChecker)],
-  });
+    afterDeclarations: [],
+  };
+
+  if (config.transformAliasedImportPaths) {
```

Comment:
Can we add this to [`src/compiler/transpile/transpile-module.ts#transpileModule`](https://github.com/ionic-team/stencil/blob/main/src/compiler/transpile/transpile-module.ts#L121) as well for single string compilation?

Reply:
Sure, that makes sense to me